src/Context/FireBase.ts

```diff
@@ -0,0 +1,20 @@
+import firebase  from "firebase/app";
+import "firebase/app";
+import "firebase/firestore";
+
+
+
+const  firebaseConfig = {
+    apiKey: "AIzaSyChkx-DzcFhTgQXdLZTIr5gHrUb3uBA_eE",
```

Comment:
These can be moved to .env
But not necessary, it's not that much of a security risk.